.firebase/hosting.YnVpbGQ.cache

```diff
@@ -0,0 +1,19 @@
+asset-manifest.json,1581546222749,53097c3f66b5f20814be9ede42ff71818522dbac9188c93725121217501d72fd
+favicon.ico,1581525961208,a08fa4488c3ecef62d9effd03b3a989929bdcbecf5e905941f9034a15bd3dba3
+index.html,1581546222749,cb0251e37c756cddccfb222e2d6876619e2fee1080483d37e5f92ceedcba70d7
```

Comment:
let's add:
```./firebase```
to .gitignore file in order to not upload this file to source control